shell/artifacts/Social/Post.schema

```diff
@@ -16,6 +16,11 @@ schema Post
   // An optional image associated with the message content for the post.
   // Populated only if the user chose to attach one when creating/editing.
   URL image
+  // A recipe that can be used to display this post's content. Allows for
+  // polymorphic display of and interaction with the post, as the
+  // particles creating or updating the Post entity can set whatever
+  // recipe they deem appropriate for the Post (or Post subclass).
+  Text recipe
```

Comment:
I considered `renderRecipe`, and had a moment of concern that using this keyword may be overloaded or have other unforeseen existing ramifications, but then, I just carried on living my life: https://www.youtube.com/watch?v=1ToyM9dhl7g

But I am open to naming feedback!